ios/engine/KMEI/KeymanEngine/Classes/Resource Management/ResourceFileManager.swift

```diff
@@ -0,0 +1,129 @@
+//
+//  ResourceManager.swift
+//  KeymanEngine
+//
+//  Created by Joshua Horton on 12/18/19.
```

Comment:
This date makes my eyes bleed 👁🩸😂.

Always one for the important code review comments.